src/hint_processor/builtin_hint_processor/math_utils.rs

```diff
@@ -117,41 +117,49 @@ pub fn assert_le_felt(
         .ok_or_else(|| HintError::MissingConstant(Box::new(PRIME_OVER_2_HIGH)))?;
     let a = &get_integer_from_var_name("a", vm, ids_data, ap_tracking)?
         .clone()
-        .into_owned();
+        .into_owned()
+        .to_biguint();
```

Comment:
I'm not sure we need all this chain. Doesn't simply `.to_biguint()` do the work here?